Tests/Features/Page/error.feature

```diff
@@ -0,0 +1,19 @@
+@mink:selenium2 @alice(Page) @alice(ErrorPage) @reset-schema
+Feature: Page not found
+
+Background:
+    Given I maximize the window
+    And I am on homepage
+
+  Scenario: I cannot acces a non exitant page
+    And I am on "/fr/imaginary-page"
+    Then I should see "404 not found"
```

Comment:
Text search is case-sensitive since #793, so you have to write `404 Not Found`, as in this change: https://github.com/Victoire/victoire/commit/0eb9d6798fbe4a6665de482eb56c63ecc657389e#diff-eb4840970f81a46a803c87a8807a7c97L67